### DISCUSSION TITLE
Access Settings via Web UI

### DISCUSSION THREAD

**Discussion by Narflar** [qdm12/ddns-updater#296] [Ideas]
Since there is a config file, how much effort would it be to have the config updated via the web UI instead of having to go into the config.json manually?

**Comment by qdm12** [comment]
I need a rather large refactor to be able to live patch settings. The program isn't really designed for that yet. I'll get to it though, but I'm rather time-bound with other repos and full time job 😿  So it'll at least be a few months to be honest.
